### DISCUSSION TITLE
Accessing browser api from pyodide

### DISCUSSION THREAD

**Discussion by TobiasHT5** [pyodide/pyodide#3651] [Q&A] [answered]
Is it possible to access the browser api from the pyodide js module? I saw it's possible to access the DOM but I don't know to what extent we can access JS APIs from the browser environment.
In particular, I wanted to access the download API in the browser.

**Comment by hoodmane** [comment, marked as answer]
`js` points to the JavaScript global scope so you can access anything in the global scope -- standard library built-ins, web apis if you're in a browser, the DOM if you're in the main thread, and any custom functions or objects that you've added to the global scope.
